### PR TITLE
Yet Another Clothing PR

### PR DESCRIPTION
**Pull request by ParadoxSpace**
- rscadd: "Adds two kinds of recolorable pants, dress and striped, as well as a recolorable beret and an alt version of the casual skirt."
  - rscadd: "Adds a non la conical hat."
  - rscadd: "You can now get an armorless version of the TCFL flight jacket in the loadout."

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 1]
```diff
@@ -213,6 +214,18 @@
 	pants["khaki shorts shorts"] = /obj/item/clothing/under/shorts/khaki/female
 	gear_tweaks += new/datum/gear_tweak/path(pants)
 
+/datum/gear/uniform/colorpants
+	display_name = "pants (recolorable)"
```
Comment (KasparoVy) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm]: Mr. Fancy Pants

**Comment by JohnBigless** [comment]
> Recolorable beret
Yes please. Fixed berets please.
+1

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -308,6 +308,12 @@
 	armor = list(melee = 40, bullet = 10, laser = 20, energy = 10, bomb = 30, bio = 0, rad = 0)
 	siemens_coefficient = 0.35
 
+/obj/item/clothing/suit/storage/toggle/leather_jacket/flight/legion/alt
+	name = "tcfl flight jacket"
```
Comment (fernerr) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
```
Unneccesary when the name is the same as the parent.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -308,6 +308,12 @@
...
+	desc = "A Tau Ceti Foreign Legion pilot's jacket. This is the more common, less durable variety, which typically finds itself percolating amongst all ranks of the TCFL."
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 10, rad = 0)
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/miscellaneous.dm]: It'd be far less snowflake to just invert the types.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 4]
```diff
@@ -308,6 +308,12 @@
...
+	siemens_coefficient = 0.75
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/miscellaneous.dm]: As above.

File: code/modules/clothing/suits/miscellaneous.dm (hunk 3 above)
Comment (ParadoxSpace) [reply code/modules/clothing/suits/miscellaneous.dm]: what

Comment (fernerr) [reply code/modules/clothing/suits/miscellaneous.dm]: I believe he means that it'd be preferable if flight/legion was the standard one, and flight/legion/alt be the armored one.

Comment (ParadoxSpace) [reply code/modules/clothing/suits/miscellaneous.dm]: oh. yeah, probably.

**Comment by ToasterStrudes** [comment]
Finally recolorable berets

**Pull request merged**